modules/deploy/pages/deployment-option/cloud/vpc-byo-gcp.adoc

```diff
@@ -0,0 +1,472 @@
+* `monitoring.timeSeries.create`
+* `monitoring.alertPolicies.get`
+* `monitoring.alertPolicies.list`
+* `monitoring.dashboards.get`
+* `monitoring.dashboards.list`
+* `monitoring.groups.get`
+* `monitoring.groups.list`
+* `monitoring.metricDescriptors.get`
+* `monitoring.metricDescriptors.list`
+* `monitoring.monitoredResourceDescriptors.get`
+* `monitoring.monitoredResourceDescriptors.list`
+* `monitoring.notificationChannelDescriptors.get`
+* `monitoring.notificationChannelDescriptors.list`
+* `monitoring.notificationChannels.get`
+* `monitoring.notificationChannels.list`
+* `monitoring.publicWidgets.get`
+* `monitoring.publicWidgets.list`
+* `monitoring.services.get`
+* `monitoring.services.list`
+* `monitoring.slos.get`
+* `monitoring.slos.list`
+* `monitoring.snoozes.get`
+* `monitoring.snoozes.list`
+* `monitoring.timeSeries.list`
+* `monitoring.uptimeCheckConfigs.get`
+* `monitoring.uptimeCheckConfigs.list`
+* `cloudnotifications.activities.list`
+* `opsconfigmonitoring.resourceMetadata.list`
+* `resourcemanager.projects.get`
+* `stackdriver.projects.get`
+* `stackdriver.resourceMetadata.list`
+* `stackdriver.resourceMetadata.write`
+* `dns.changes.create`
+* `dns.changes.get`
+* `dns.changes.list`
+* `dns.managedZones.list`
+* `dns.resourceRecordSets.create`
+* `dns.resourceRecordSets.delete`
+* `dns.resourceRecordSets.get`
+* `dns.resourceRecordSets.list`
+* `dns.resourceRecordSets.update`
+* `secretmanager.versions.access`
+* `storage.objects.get`
+* `storage.objects.list`
+====
+
+... Project bindings
+
+* GKE custom role
+
+.. Redpanda Console SA
+
+... Redpanda Console custom role permissions
++
+.Expand necessary permissions
+[%collapsible]
+====
+* `secretmanager.secrets.create`
+* `secretmanager.secrets.delete`
+* `secretmanager.secrets.list`
+* `secretmanager.secrets.update`
+* `secretmanager.versions.add`
+* `secretmanager.versions.destroy`
+* `secretmanager.versions.disable`
+* `secretmanager.versions.enable`
+* `secretmanager.versions.list`
+* `iam.serviceAccounts.getAccessToken`
+
+NOTE: If `iam.serviceAccounts.getAccessToken`` is not added, there will be errors in the Redpanda Console pod log.
+====
+
+... Project bindings
+
+* Redpanda Console custom role
+
+.. Redpanda Connectors SA
+
+... Connectors custom role permissions
+
+* `resourcemanager.projects.get`
+* `secretmanager.versions.access`
+
+... Project bindings
+
+* Connectors custom role
+
+. Because some resources can only be created after the Redpanda ID is known, xref:./create-byoc-cluster-gcp.adoc[create a BYOC cluster] in the https://cloudv2.redpanda.com[Redpanda Cloud UI^] to get the Redpanda ID.
++
+IMPORTANT: On the Network page, select the *Customer-managed* network connection type. Redpanda validates your network, service account, and storage bucket configuration. Click *Next*, but before running the `rpk` command on the Deploy page of the UI, note the redpanda-id (for example, cisld88gfi809ee1qjcg). You finish the cluster deployment in a later step.  
```

Comment:
We don't validate on this page, probably in the future there will be a command provided so that the user can validate on their own.